src/components/CreateTeam/container.js

```diff
@@ -15,8 +15,11 @@ export default Wrapped =>
     }
 
     state = {
-      teamSubmitted: false,
       clients: [],
+      initialFormValues: {
+        selectedClient: '-1',
+        teamName: '',
+      },
```

Comment:
I don't think this belongs in state. The initial values will never change. That or they're doing more than being initial values.

Reply:
Yeah, the naming is misleading here. I've updated initialFormValues to formValues. The state is used as it is required to update the formik state. This is done once a response comes back from a successful createTeam post.